src/build/buildNativefierApp.ts

```diff
@@ -65,7 +65,18 @@ async function copyIconsIfNecessary(
     options.packager.platform === 'darwin' ||
     options.packager.platform === 'mas'
   ) {
-    log.debug('No copying necessary on macOS; aborting');
+    if (options.nativefier.tray) {
+      //tray icon needs to be .png
+      log.debug('Copying icon for tray application');
+      const trayIconFileName = `icon.png`;
```

Comment:
I think we should probably pick a different filename here as this could conflict with a linux icon file.

Reply:
What about tray-icon?

Reply:
Works for me.